packages/vira-book/src/element-book/entries/dropdown/vira-dropdown-item.element.book.ts

```diff
@@ -0,0 +1,86 @@
+import {BookPageControlTypeEnum, defineBookPage, definePageControl} from 'element-book';
+import {CSSResult, HTMLTemplateResult, html} from 'element-vir';
+import {ViraDropdownItem} from 'vira';
+import {dropdownPage} from './vira-dropdown.book';
+
+const examples: ReadonlyArray<{
+    title: string;
+    inputs?: typeof ViraDropdownItem.inputsType;
+    customStyle?: CSSResult;
+    customTemplate?: HTMLTemplateResult;
+}> = [
+    {
```

Comment:
Please add examples: 
- having not enough horizontal space
- being stretched to take up more space than necessary